harness/determined/cli/config_policies.py

```diff
@@ -91,20 +140,28 @@ def delete_config_policies(args: argparse.Namespace) -> None:
                     cli.Arg(
                         "workload_type",
                         type=str,
-                        choices=["experiment", "ntsc"],
-                        help="the type (Experiment or NTSC ) of config policies",
+                        choices=["experiment", "tasks"],
+                        help="the type (Experiment or Tasks) of config policies",
                     ),
                     cli.Arg(
-                        "--workspace",
+                        "config_file",
                         type=str,
-                        required=True,  # change to false when adding --global
-                        help="config policies for the given workspace",
+                        help="path to the YAML or JSON file containing defined config policies "
+                        "(can be absolute or relative to the current directory)",
                     ),
-                    cli.Arg(
-                        "--config-file",
-                        type=str,
-                        required=True,
-                        help="path to the yaml file containing defined config policies",
+                    cli.Group(
+                        cli.Arg(
+                            "-w",
+                            "--workspace-name",
+                            type=str,
+                            help="apply config policies to this workspace",
+                        ),
+                         cli.Arg(
```

Comment:
I thought that if a workspace wasn't given, we were going to use the global APIs?

Reply:
Gotcha! I was under the impression we wanted a flag for global based on [these](comments)

Reply:
I'll remove the flag

Reply:
Hmm, the link isn't working for me. What do you think makes more sense / is more intuitive as a user? For me, I don't want to have to specify a global flag. If I don't put a workspace, then I assume it applies to the global scope.

I do remember having conversations about not inferring global so that users don't make mistakes (aka they forget the workspace field and now it's global). I'm not particularly worried about that; only admins can update it and they'll be using the webui most of the time.

Reply:
If you feel strongly that there should be a global flag, we can talk about it though!

Reply:
> I do remember having conversations about not inferring global so that users don't make mistakes 

This was my hesitation with not adding a global flag~ But yea I agree, this is a probably an unlikely use case, and I think the CLI commands are cleaner without it!

Also, just for ref, [this](https://github.com/determined-ai/determined/pull/9911/files#diff-499099d1c98bc0dbc075690cd3985efc2977ea57c5a5e1559e8398a0af5486deR79) is what i was referring to (I think my original link got wiped away with a rebase lol)